sdk/src/erc20.ts

```diff
@@ -6,93 +6,141 @@ import {
   CasperClient,
   CasperServiceByJsonRPC,
   Contracts,
-  DeployUtil,
   Keys,
   RuntimeArgs
 } from 'casper-js-sdk'
-import { getWasm } from './utils'
+import { BALANCES, DEFAULT_PAYMENT_AMOUNT } from './consts'
+import { Network } from './network'
+import { getDeploymentData, hash, hexToBytes, sendTx } from './utils'
+
+const CONTRACT_NAME = 'erc20'
 
 export class Erc20 {
```

Comment:
You were using legacy schema for `deploying`/`loading` in Erc20. We have modified it to support multiple contracts within a single instance of `Erc20`. You can now use the `setContract` function to select the target contract.